package.json

```diff
@@ -36,7 +36,7 @@
     "body-parser": "^2.0.1",
     "content-disposition": "^1.0.0",
     "content-type": "~1.0.4",
-    "cookie": "0.7.1",
+    "cookie": "1.0.1",
```

Comment:
@blakeembrey can you comment on this? I believe the breaking changes in 1.x are passed through and so must go out with a major of express, but maybe I am wrong.